### PR TITLE
Add a separate introduction page

### PR DESCRIPTION
**Pull request by gjvis**

### CODE REVIEW

File: app/views/searches/new.html.erb [hunk 1]
```diff
@@ -1,21 +1,11 @@
-<% content_for(:page_title_prefix, "Search") %>
+<% content_for(:page_title_prefix, "Find information on roles") %>
 
-<h2 class="heading-xlarge">Explore work you could do</h2>
+<h2 class="heading-xlarge">Find information on roles</h2>
 
 <% if current_scrapbook.saved_occupations.any? %>
```
Comment (lukasz-kaniowski) [inline app/views/searches/new.html.erb]: `current_scrapbook.saved_occupations.any?` logic duplicated from controller. Wouldn't be better to pass this as a model to view, i.e. `showSavedRoles?`

Comment (gjvis) [inline app/views/searches/new.html.erb]: `SearchesController#new` doesn't have any logic in it, so this isn't duplicated anywhere. You're right that its a bit dirty doing it in the view though – we could move it to the controller? (This hasn't changed as part of this PR though)

Comment (lukasz-kaniowski) [inline app/views/searches/new.html.erb]: you are right. `ScrapbooksController` has very similar code and I simply assume that this is the corresponding controller. my bad. I'm not sure what is the ROR way of doing this.

**Comment by lukasz-kaniowski** [comment]
looks good 👍

**Pull request merged**